packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -219,6 +219,14 @@ export const createZipClicked: Action = ({ state, effects }) => {
   effects.zip.download(state.editor.currentSandbox);
 };
 
+export const forkExternalSandbox: AsyncAction<{
+  id: string;
+}> = async ({ actions }, { id }) => {
```

Comment:
Since we do not have Cerebral anymore we are free to type payload as normal values :) That means you can express this as:

```ts
export const forkExternalSandbox: AsyncAction<string> = ({ actions}, id) {...}
```